packages/runtime/test/src/interpreter/opcode-list.ts

```diff
@@ -3098,4 +3098,117 @@ describe("Teal Opcodes", function () {
       );
     });
   });
+
+  describe("PushInt", () => {
+    let stack: Stack<StackElem>;
+    this.beforeEach(() => { stack = new Stack<StackElem>(); });
+
+    it("should push uint64 to stack", () => {
+      const op = new PushInt([MAX_UINT64.toString()], 0);
+      op.execute(stack);
+
+      assert.equal(1, stack.length());
+      assert.equal(MAX_UINT64, stack.pop());
+    });
+  });
+
+  describe("PushBytes", () => {
+    let stack: Stack<StackElem>;
+    this.beforeEach(() => { stack = new Stack<StackElem>(); });
+
+    it("should push bytes to stack", () => {
+      const str = "\"Algorand\"";
+      const bytes = new Uint8Array(Buffer.from("Algorand"));
+
+      const op = new PushBytes([str], 1);
+      op.execute(stack);
+
+      assert.equal(1, stack.length());
+      assert.deepEqual(bytes, stack.pop());
+    });
+  });
+
+  describe("Assert", () => {
+    let stack: Stack<StackElem>;
+    this.beforeEach(() => { stack = new Stack<StackElem>(); });
+
+    it("should not panic if top of stack is non zero uint64", () => {
+      const op = new Assert([], 0);
+      stack.push(55n);
+      assert.doesNotThrow(() => { op.execute(stack); });
+    });
+
+    it("should panic if top of stack is not valid", () => {
+      const op = new Assert([], 0);
+      stack.push(0n);
+
+      expectRuntimeError(
+        () => op.execute(stack),
+        RUNTIME_ERRORS.TEAL.TEAL_ENCOUNTERED_ERR
+      );
+
+      stack.push(stringToBytes("HelloWorld"));
```

Comment:
is this case verified by running on node ?

Reply:
yes. It's also mentioned here that `assert` pops `unit64` from stack: [https://developer.algorand.org/docs/reference/teal/opcodes/#assert](https://developer.algorand.org/docs/reference/teal/opcodes/#assert)